packages/cache-inmemory/jest.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  globals: {
+    "ts-jest": {
+      tsConfig: "tsconfig.json",
```

Comment:
question: Does this works? Since this config exclude all tests files? Should we have a separate tsconfig.spec.json as we do in the plugins?

Reply:
If it works, let's keep it that way, I would have just thought that since they are excluded in the tsconfig it might be a problem 😂 Let's keep it that way and see later to harmonised the different plugins and modules as they have or not a tsconfig.spec depending.